packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -43,11 +43,7 @@ function SplashHandler({
   };
 
   useEffect(() => {
-    if (!isContentVisible) {
-      return;
-    }
-
-    const updateWithErrorLog = (prevIsSplashOn: boolean) => {
+    const setFalseWithErrorLog = (prevIsSplashOn: boolean) => {
```

Comment:
Now that I think about it, I'd find it more readable if this function was inlined in `setIsSplashOn()` - it's not complicated enough to have its readability benefit frome extracting it, yet when it's extracted it forces the reader to jump through the code. Just a suggestion.